config.sample.toml

```diff
@@ -182,7 +182,12 @@ port = 4000
 # Time Interval (in seconds) in which metrics of all application containers
 # running in the current node are collected and stored in the central mongoDB database
 metrics_interval = 600
-
+# Limits the number of app instances that can be deployed by an user
+# Set app_limit = -1 if no limit is to be imposed
+app_limit = 10
+# Time interval (in hours) for rate limiting
```

Comment:
The comment is not really informative. We should also inform that x instances can be deployed in this time interval but the total number of instances should not be more than `app_limit`